src/store/tbtc/tbtcSlice.ts

```diff
@@ -18,6 +18,7 @@ export const tbtcSlice = createSlice({
   name: "tbtc",
   initialState: {
     mintingStep: MintingStep.ProvideData,
+    depositStep: "bitcoin-confirmations",
```

Comment:
Can we name it slightly different? `mintingStep` and `depositStep` assosiate with the same thing, and for me, the first thought was the deposit flow before the reveal. We use both terms alternately for that, although we shouldn't, because I think the `mintingStep` would be more correct to use for the reveal flow that happens on the bridge, and `depositStep` would be better for the user deposit flow. We don't want to refactor that now though.

I would propose `depositDetailsStep` (or alternatively - `revealDepositStep` but I think the first one is better). It would clearly indicate, that this step is not related to the user minting flow.

Reply:
Ref commit: ca483030